codespell_lib/data/dictionary.txt

```diff
@@ -8790,7 +8790,11 @@ corrolation->correlation
 corrolations->correlations
 corrrect->correct
 corrrected->corrected
+corrrecting->correcting
+corrrection->correction
```

Comment:
Please also add `corrrections->corrections`

Reply:
Added it. https://github.com/codespell-project/codespell/blob/e564aa64a22def72e952df8dc08c96f5e809ff20/codespell_lib/data/dictionary.txt#L8795